alembic/versions/5b141f32ea38_remove_bad_autoincrements.py

```diff
@@ -0,0 +1,42 @@
+"""remove bad autoincrements
+
+Revision ID: 5b141f32ea38
+Revises: 4653b4902dc0
+Create Date: 2017-07-27 22:51:11.172610+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.schema import Sequence, CreateSequence
+
+
+# revision identifiers, used by Alembic.
+revision = '5b141f32ea38'
+down_revision = '4653b4902dc0'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Should ditch the `###` comments in these files if/when the commands have been vetted by a human.